spectest/shared/phase0/epoch_processing/helpers.go

```diff
@@ -62,8 +61,6 @@ func RunEpochOperationTest(
 		pbState, err := v1.ProtobufBeaconState(beaconState.InnerStateUnsafe())
 		require.NoError(t, err)
 		if !proto.Equal(pbState, postBeaconState) {
-			diff, _ := messagediff.PrettyDiff(beaconState.InnerStateUnsafe(), postBeaconState)
```

Comment:
I find this not very helpful, when it happens it usually logs out 10000 lines of proto fields and a few lines of state fields which gets lost